File: Up_wpf/Up_wpf/Group.cs
using System;
using System.Collections;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Up_wpf
{
    public class Group
    {
        private string name;
        private ArrayList up_rasp;
        private ArrayList down_rasp;

        public Group()
        {
            up_rasp = new ArrayList();
            down_rasp = new ArrayList();
        }
        public void setName(string name)
        {
            this.name = name;
        }

        public string getName()
        {
            return name;
        }
        public void up_add(ArrayList days)
        {
            up_rasp.AddRange(days);

        }
        public void down_add(ArrayList days)
        {
            down_rasp.AddRange(days);

        }
        public ArrayList getDown_rasp()
        {
            return down_rasp;
        }

        public ArrayList getUp_rasp()
        {
            return up_rasp;
        }
    }
}

File: Up/app/src/main/java/com/product/dstavrov/up/Helpers/Group.java
package com.product.dstavrov.up.Helpers;

import java.util.ArrayList;

/**
 * Created by vicru on 20.03.2018.
 */

public class Group {
    private String name;
    private ArrayList<Day> up_rasp;
    private ArrayList<Day> down_rasp;

    public Group(){
        up_rasp = new ArrayList<Day>();
        down_rasp = new ArrayList<Day>();
    }
    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
    public void up_add(ArrayList<Day> days){
        up_rasp.addAll(days);

    }
    public void down_add(ArrayList<Day> days){
        down_rasp.addAll(days);

    }

    public ArrayList<Day> getDown_rasp() {
        return down_rasp;
    }

    public ArrayList<Day> getUp_rasp() {
        return up_rasp;
    }
}
